packages/xrpl/src/models/transactions/metadata.ts

```diff
@@ -40,6 +40,7 @@ export interface DeletedNode {
   DeletedNode: {
     LedgerEntryType: string
     LedgerIndex: string
+    PreviousFields?: { [field: string]: unknown }
```

Comment:
I couldn't find the `PreviousFields` sub-section under `DeletedNode`. It only lists `LedgerWntryType`, `LedgerIndex` and `FinalFields`
Reference: https://xrpl.org/docs/references/protocol/transactions/metadata/#deletednode-fields

Only `ModifiedNode` has a sub-object titled `PreviousFields`

Reply:
It's listed in the transaction linked above. The docs are incorrect then.

Reply:
Okay, I couldn't find the specific example that you are referring to -- but I agree with your premise. I verified in the code.

Reply:
All the `PreviousFields` appear under `ModifiedNode` in the first example of the docs page

Reply:
This is the link in the PR description above: https://livenet.xrpl.org/transactions/424661CF1FD3675D11EC910CF161979553B6D135F9BD03E6F8D4611D88D27581/raw
The first deleted node has `PreviousFields` and `FinalFields`

Reply:
oh, I thought you were referring to the previous transaction in the docs link. Thanks, this clears it up